wandb/fastai/__init__.py

```diff
@@ -212,3 +218,10 @@ def on_train_end(self, **kwargs):
                     self.learn.load(model_file, purge=False)
                     print('Loaded best saved model from {}'.format(
                         self.model_path))
+
+        if self.confusion_matrix:
+            interpret = ClassificationInterpretation.from_learner(self.learn)
+            conf_plt = interpret.plot_confusion_matrix(dpi=self.conf_dpi, return_fig=True, title='Confusion Matrix')
+            self.interpret = interpret # save for later use if needed
```

Comment:
When called outside training, we need to call `learn.interpret()` again. However, storing as `self.interpret = interpret()` gives an error.